### ISSUE TITLE
Deck Calibration: Calibrate XY Point 'C'

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#981]
As a robot user, I would like to be able to calibrate my robot's X and Y axis. This story covers calibrating point 'B'

## Acceptance Criteria
_After user clicks 'Save and Move to Next Point' for point 'B' and pipette moves to point 'C' on the deck (US #980 )_:
- User prompted to jog pipette until the tip is directly over point 'C'
-- User given visual aids for correct positioning
-- User given full jog controls (X, Y, Z, and speed control)
-- User able to jog via keyboard shortcuts or app buttons
- User has button to 'Save'
-- Clicking button saves Point 'C' calibration
-- User receives confirmation that point is saved
-- User instructed to remove tip
- User has a 'Save Calibration' button
-- Button functionality in separate story

## Design
- [Invision](https://projects.invisionapp.com/d/main#/console/13450400/280769863/preview)

## Implementation Notes
- Dependent on #951 for API function to move to XY calibration points
- Dependent on #952 for API function to save XY calibration points
